Builder/Exception/MissingAssociationAdmin.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Builder\Exception;
+
+/**
+ * Signals that a field description is missing an association admin when a form type requires it.
+ *
+ * @author ju1ius <ju1ius@laposte.net>
+ */
+final class MissingAssociationAdmin extends \LogicException
```

Comment:
Yeah so tell me why PHP named all built-in exception `*Exception`?

> Also, when using many exception, one of my colleagues recommends doing this:

This works when all your exception come form the same namespace.

And only for one it's still:

```php
throw new ThisShouldNotHappen();
```

Anyway, Symfony **and** PHP works like this, so I don't see any reason to not do the same.